app/assets/javascripts/angular/controllers/acts.js.coffee

```diff
@@ -1,16 +1,68 @@
-@ActsCtrl = ['$scope', '$http', ($scope, $http) ->
+@ActsCtrl = ['$q', '$scope', '$http', '$translate', '$cookies', 'ClientActs', ($q, $scope, $http, $translate, $cookies, ClientActs) ->
   $scope.actParams = {
     month: moment().format('MM/YYYY')
   }
 
+  $scope.changeSelect = () ->
+    $scope.currentCustomer = _.find $scope.customers, (customer) ->
+      parseInt($scope.actParams.customer_id) == customer.id
+    $scope.acts = ClientActs.query(
```

Comment:
it's not necessary to use parentheses
+    $scope.acts = ClientActs.query
 +      customer_id: $scope.currentCustomer.id
 +      () ->
 +        $("#generated-act").html ''
 +        $scope.showCurrentAct = false
 +        $scope.infoActEmpty = $scope.acts.length == 0
 and if you use underscore.js so you should  $scope.infoActEmpty = _.isEmpty $scope.acts